content/applications/general/email_communication/faq.rst

```diff
@@ -72,7 +76,7 @@ If the daily limit is reached:
 SMTP error
 **********
 
-Simple Mail Transport Protocol (SMTP) error messages explain why an email wasn't transmitted
+Simple Mail Transport Protocol (SMTP) error messages explain why an email was not transmitted
```

Comment:
```suggestion
*Simple Mail Transport Protocol (SMTP)* error messages explain why an email was not transmitted
```
suggesting the use of italics here because this is, essentially, a definition.